src/ext/SandboxFeature.tsx

```diff
@@ -43,10 +43,10 @@ const SandboxFeature = <Features, Props extends SandboxFeatureProps<Features>>({
     throw new Error('Max feature depth is 2');
   } else if (splitFeature.length === 2) {
     Component = memo(
-      lazy(() => import(`./${splitFeature[0]}/${splitFeature[1]}/index.ts`)),
+      lazy(() => import(`./${splitFeature[0]}/${splitFeature[1]}/index.tsx`)),
```

Comment:
Should there be an additional prop like "type" or "extension" as part of this, so that this becomes dynamic?
```
export interface SandboxFeatureProps<Features> {
  feature: keyof Features;
  renderFallback?: () => ReactNode;
}
```

Reply:
In some cases you maybe want to export several types from the same file. I think we can just standardize on .tsx since it will always work and it allows to write JSX directly in the index file if you want to do that. My original idea is that the index file would only re-export from other files, hence it was just a .ts.